### PR TITLE
BETA: Register lykcloud.is-a.dev

### PR DESCRIPTION
**Pull request by YashRawat911**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [] The CNAME record doesn't contain `https://` or `/`.  
- [] There is sufficient information at the `owner` field.  

## Website Link/Preview

lykcloud.zc.al

### CODE REVIEW

**Review by priyazsh — APPROVED** [review]
LGTM

**Review by YashRawat911 — COMMENTED** [review]
Ok

**Comment by YashRawat911** [comment]
Update

**Review by YashRawat911 — COMMENTED** [review]
Update

File: domains/lykcloud.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "LykCloud",
```
Comment (oyepriyansh) [inline domains/lykcloud.json:3]: The username must be the same as your GitHub username.

**Review by priyazsh — CHANGES_REQUESTED** [review]

File: domains/lykcloud.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "YashRawat911",
+        "email": "lykcloud@dr.com"
+    },
+    "record": {
+        "CNAME": "f5023159-9ff2-4823-9493-d8eae763f6dd.id.repl.co",
+        "TXT": "replit-verify=f5023159-9ff2-4823-9493-d8eae763f6dd"
```
Comment (WilliamDavidHarrison) [inline domains/lykcloud.json:8]: We do not support TXT verification.

File: domains/lykhost.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "YashRawat911",
+        "email": "911t.empm.ail@gmail.com"
+    },
+    "record": {
+        "CNAME": "lykcloud.zc.al"
```
Comment (WilliamDavidHarrison) [inline domains/lykhost.json:7]: We do not allow free hosting services to use our subdomains.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**